nvm.sh

```diff
@@ -1789,7 +1789,7 @@ nvm_get_make_jobs() {
   elif [ "_$NVM_OS" = "_sunos" ]; then
     NVM_CPU_THREADS="$(psrinfo | wc -l)"
   elif [ "_$NVM_OS" = "_aix" ]; then
-    NVM_CPU_THREADS="$(lsconf | command grep 'Number Of Processors:'| command awk '{print $4}')"
+    NVM_CPU_THREADS="$(pmcycles -m | wc -l)"
```

Comment:
Right, but the purpose of this check is to ensure that there is always at least one *CPU* available to process commands besides `nvm install -s`, to avoid locking up the machine. Using the thread count will not ensure there is always at least one processor available.